insights/parsers/softnet_stat.py

```diff
@@ -0,0 +1,100 @@
+"""
+softnet_stat - /proc/net/softnet_stat
+=====================================
+
+This parser parses the stats from network devices. These stats includes events
+per cpu(in row), number of packets processed i.e packet_process (first column),
+number of packet drops packet_drops (second column), time squeeze i.e net_rx_action
+performed time_squeeze(third column), cpu collision i.e collision occur while obtaining
+device lock while transmitting cpu_collision packets (Ninth column), received_rps
+number of times cpu woken up received_rps (tenth column), number of times reached flow
+limit count flow_limit_count (eleventh column).
+
+    The typical contents of ``cat /proc/net/softnet_stat`` file is as below::
+
+        00008e78 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        000040ee 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        0001608c 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        0000372f 00000001 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+
+        Column-01: packet_process: Packet processed by each CPU.
+        Column-02: packet_drop: Packets droped.
+        Column-03: time_squeeze: net_rx_action.
+        Column-09: cpu_collision: collision occur while obtaining device lock while transmitting.
+        Column-10: received_rps: number of times cpu woken up received_rps.
+        Column-11: flow_limit_count: number of times reached flow limit count.
+
+    Examples:
+        >>> SOFTNET_STATS = '''
+        ... 00008e78 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        ... 000040ee 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        ... 0001608c 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        ... 0000372f 00000001 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        ...'''.strip()
+        >>> softnet_obj = SoftNetStats(context_wrap(SOFTNET_STATS))
+        >>> softnet_obj.is_packet_drops()
+        True
+        >>> softnet_obj.cpu_instances
+        4
+        >>> softnet_obj.per_cpu_nstat('packet_drops')
+        [0, 0, 0, 1]
+"""
+
+from .. import Parser, parser
+
+
+@parser("softnet_stat")
+class SoftNetStats(Parser):
+    """Parses ``/proc/net/softnet_stat`` file contains"""
```

Comment:
Please include docstrings for the attributes `cpu_instances` and `nstat_per_cpu`